clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesServerGroup.groovy

```diff
@@ -120,14 +170,11 @@ class KubernetesServerGroup implements ServerGroup, Serializable {
     this.events = events?.collect {
       new KubernetesEvent(it)
     }
-    if (autoscaler) {
```

Comment:
I think we still need this here. There's only 1 place in this diff where it gets attached, but it's in the `buildCacheResult` which leads me to believe that this will cause a regression with respect to HPSs. @lwander double check me on that, please.